modules/update-mirror-repository.adoc

```diff
@@ -98,6 +98,18 @@ $ oc adm release mirror -a ${LOCAL_SECRET_JSON} --to-dir=${REMOVABLE_MEDIA_PATH}
 ----
 . Mirror the version images to the mirror registry.
 ** If your mirror host does not have internet access, take the following actions:
+... If the local container registry and the cluster are connected to the mirror host, directly push the release images to the local registry and apply the config map  to the cluster by using following command:
```

Comment:
`--from=quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE} --to=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}` is `oc` reaching out to Quay to get the release payload and reaching out to the local registry to drop off that payload where the cluster can find it.  `--apply-release-image-signature` is reaching out to [either of the signature locations][1] to pull the signature, and then reaching out to the cluster to insert a signature ConfigMap.  Neither one of the external reaches (Quay or the signature locations) will work in this case, which is `If your mirror host does not have internet access...`.  I think we want to leave the `--apply-release-image-signature` content where it is today.

The bit we do want here [the *Creating the config map for image signature verification by using the oc CLI* content][2] (which would go at the end of this `If your mirror host does not have internet access...` branch, right after the `oc image mirror ...` call pushes the sneakernetted release payload up into the local registry) to push the signature from the sneakernetted `mirror/config/signature-sha256-*.yaml` up into the local cluster.

[1]: https://github.com/openshift/cluster-update-keys/blob/342eb8e2631f3218ea386479102784d5e288d868/manifests.rhel/0000_90_cluster-update-keys_configmap.yaml#L4-L5
[2]: https://github.com/openshift/openshift-docs/blob/d4762f0f626a4dddb9d7330e63a3bb6cb73f5bb5/modules/update-oc-configmap-signature-verification.adoc

Reply:
Agree with @wking, this change definitely will break the original working procedure.